src/Classes/ServiceAPI/MyRadio_Podcast.php

```diff
@@ -605,7 +611,7 @@ public function setSuspended(bool $is_suspended)
      *
      * @param MyRadio_Show $show
      */
-    public function setShow(MyRadio_Show $show)
+    public function setShow($show)
```

Comment:
Why this change (and subsequent `if ($show instanceof MyRadio_Show) {`) ?

Reply:
yeah no idea about this @mstratford mind clarifying?

Reply:
Erm, I'm not sure either, hmm

Reply:
It would be because $show can be null (if no show has been selected for the podcast), and PHP doesn't treat that as an acceptable MyRadio_Show object.